test/e2e/framework.go

```diff
@@ -726,6 +726,22 @@ func (data *TestData) podWaitForIP(timeout time.Duration, name, namespace string
 	return pod.Status.PodIP, nil
 }
 
+// podWaitForNode polls the K8s apiserver until the specified Pod is in the "running" state (or until
+// the provided timeout expires). The function then returns the node which the Pod is scheduled on.
+func (data *TestData) podWaitForNode(timeout time.Duration, name, namespace string) (string, error) {
```

Comment:
I don't think you ended up using this, you went with `k8sUtils.GetPod` :)

Reply:
Yes, I mentioned this in the PR comment. The actual Pods (x/a etc.) are suffixed with the replicaSet random strings. I figured this util function might still be useful in future occasions... But since adding this is a breeze, will remove it from this particular PR.